multi-node/aws/pkg/cluster/cloud_config_controller.go

```diff
@@ -14,11 +14,31 @@ coreos:
     initial-advertise-peer-urls: http://$private_ipv4:2380
     listen-client-urls: http://0.0.0.0:2379
     listen-peer-urls: http://0.0.0.0:2380
-    initial-cluster: controller=http://$private_ipv4:2380
 
   units:
+  - name: etcd-peers.service
```

Comment:
Given that we want to run in HA mode, then by definition it has to work in parallel on several machines. ;-)  So far we've had no issues with it, but I am guessing there may be some scenarios not handled.  Alas, I've not explored them.  Might want to ping the Monsanto folks that wrote it.  They probably have some experience running it on larger cluster for far longer and can give you a better idea about failure scenarios.